tests/integration_rust/add_tests.rs

```diff
@@ -894,30 +894,59 @@ preview = ['pixi-build']"#,
 #[tokio::test]
 async fn add_pypi_git() {
     let pixi = PixiControl::from_manifest(
-        r#"
+        format!(
+            r#"
 [project]
 name = "test-channel-change"
 channels = ["https://prefix.dev/conda-forge"]
-platforms = ["linux-64"]
+platforms = ["{}"]
```

Comment:
maybe you can use `{platform}` and `platform = Platform::current()`? If that works :)